staging/src/k8s.io/client-go/tools/clientcmd/client_config_test.go

```diff
@@ -553,3 +588,46 @@ func TestNamespaceOverride(t *testing.T) {
 
 	matchStringArg("foo", ns, t)
 }
+
+func TestAuthConfigMerge(t *testing.T) {
```

Comment:
this passes on master without the mergo bump... can we add a test that reproduces https://github.com/kubernetes/client-go/issues/431 on master?

Reply:
This is the exact test from that issue, but I can't actually reproduce https://github.com/kubernetes/client-go/issues/431 at this time. Should I just remove this test completely?

Reply:
turns out this test caught issues only present between 0.3.0 and 0.3.2. it's fine to keep it anyway